src/NuGet.Core/NuGet.Packaging/PackageCreation/Resources/NuGetResources.resx

```diff
@@ -208,4 +208,12 @@
   <data name="IconNoFileElement" xml:space="preserve">
     <value>The icon file '{0}' does not exist in the package.</value>
   </data>
+  <data name="Manifest_IconUrlCannotBeUsedWithIcon" xml:space="preserve">
+    <value>The iconUrl and icon elements cannot be used together.</value>
```

Comment:
Why the restriction? This will complicate the scenario of publishing to multiple feeds.

Reply:
I added the restriction to be consistent with licenses file behavior. You can't use `<license>` and `<licenseUrl>` at the same time in the .nuspec file.

Reply:
Discussed offline. This will be now a warning instead of a restriction/error.